docs/HowToGuides/GettingStarted.md

```diff
@@ -250,11 +250,15 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
      ```
      If you installed and want to use Sccache, include the `--sccache` option in
      the invocation as well.
+   <!-- FIXME: Without this "hard" line break, the note doesn’t get properly spaced from the bullet -->
+   <br />
 
    > **Note**  
-   > If you aren't planning to edit the parts of the compiler that are written
-   > in Swift, pass `--bootstrapping=hosttools` to speed up local development.
-   > This requires a recent Xcode and/or swift toolchain to be installed.
+   > If you aren't planning to work on the parts of the compiler that are
+   > written in Swift, pass `--bootstrapping=hosttools` or `--bootstrapping=off`
+   > to speed up local development. Note that on Linux `--bootstrapping=hosttools`
+   > requires [a recent Swift toolchain](https://www.swift.org/download/) to be
+   > installed.
```

Comment:
@eeckstein I know you removed the `off` option from here, but I feel really bad about omitting that the toolchain installation is not the only way to get rid of bootstrapping overheads on Linux.

Reply:
We will soon be removing the `off` option (not only from the docs) for linux and macos.

Reply:
I see, makes sense.